exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -266,13 +389,15 @@ void createEmptyBlock(TestKit testKit) {
 
   @Test
   void afterCommitSubmitsTransaction(TestKit testKit) {
+    int serviceId = testKit.getServiceIdByName(SERVICE_NAME);
```

Comment:
I'm not sure if we need this method though. As of now, user creates TestKit with his own serviceId, so he knows it and doesn't need to retrieve it (here we could just use `SERVICE_ID`). But this method allows us to generate serviceId inside the TestKit, so that user doesn't need to worry about it, which would be nice.

Reply:
I'd add if it becomes needed. This info can also be retrieved from the DispatcherSchema using the map we already have.

Reply:
Ok, removed it